source/contracts/Controlled.sol

```diff
@@ -20,7 +20,7 @@ contract Controlled is IControlled {
     }
 
     modifier onlyControllerCaller {
-        require(IController(msg.sender) == controller);
+        require(IController(msg.sender) == controller || controller == address(0));
```

Comment:
Can you go into detail on how this is a good thing?  It means that if the controller address is ever set to 0, any `onlyControllerCaller` calls will be callable by anyone (essentially public with no modifier).  At the least, the modifier name should be renamed to be more clear, though I would like to see some strong arguments for why this should be this way.

Reply:
Same could be said for anyone calling the initialize function on any of our factory generated contracts. Its yet another thing we need to do as part of initialization (setting the controller). We also can't overload the initialization function with this as some contracts (Market in particular) cannot have any additional variables added to their initialize method.

That said we probably don't really need the `Controlled` class as much anyway as lookups should only really be relevant at construction time and only during the initial deploy.

There will be a separate issue for evaluating / removing the `Controlled` class entirely.